pkg/retagger/job_definition.go

```diff
@@ -160,10 +160,10 @@ func fromImageTagPattern(image images.Image, tagPattern images.TagPattern) (JobD
 	j := JobDefinition{
 		SourceImage:   image.Name,
 		SourcePattern: tagPattern.Pattern,
-	}
-
-	if image.OverrideRepoName != "" {
-		j.Options.OverrideRepoName = image.OverrideRepoName
+		Options: JobOptions{
+			TagTrimVersionPrefix: image.TagTrimVersionPrefix,
+			OverrideRepoName:     image.OverrideRepoName,
```

Comment:
Is this fine if we don't check for the empty case first? I am not sure how the code is handled.

Reply:
By removing the check, when `OverrideRepoName` is the empty string, we would assign an empty string to `OverrideRepoName` which is already initialized to the zero value (no pointers involved), so I don't see how it would affect the behavior.